content/_code-samples/checks/js/cancel_check.js

```diff
@@ -0,0 +1,44 @@
+const xrpl = require('xrpl');
```

Comment:
Could you re-name these files to use `-` instead of `_` to match the other folders' naming convention? (Ex. `cancel_check.js` -> `cancel-check.js`)